### DISCUSSION TITLE
Elevation offset / Water

### DISCUSSION THREAD

**Discussion by redbu11y** [jetelain/ArmaRealMap#189] [Q&A]
So I'm trying to make a Khalkhin Gol map, and the general layout from the tool is a very good starting point. My issue is that i need water in the actual Khalkhin Gol river. Looking at the height markers in the map as well as the exported heigthmap asc file, it seems that the map is about 670 meters above sea level at its lowest point. Is there a way to offset the heigthmap to 0m?

(Will editing the asc file with a python script and importing it work?)

**Comment by redbu11y** [comment]
So editing the asc file with a python script to offset the elevation with -660 somewhat worked, but the ares out of bounds are still at their original elevation and causing the edges of the map to be a huge cliff.

![20240309050639_1](https://github.com/jetelain/ArmaRealMap/assets/17026041/711d0966-ee70-4be3-822e-39ccd683ea5b)

**Reply by katzen22** [reply]
Have you managed to find a solution to this problem?

**Reply by redbu11y** [reply]
No, i am afraid not.

**Reply by jetelain** [reply]
This is caused by out of bound elevation configuration. You can disable it to avoid this behavior. Edit `mapinfos.hpp` and set `enableTerrainSynth` to `0`.
